CRM/Case/Form/Activity/ChangeCaseStartDate.php

```diff
@@ -27,10 +27,10 @@ class CRM_Case_Form_Activity_ChangeCaseStartDate {
    */
   public static function preProcess(&$form) {
     if (!isset($form->_caseId)) {
-      CRM_Core_Error::fatal(ts('Case Id not found.'));
+      CRM_Core_Error::statusBounce(ts('Case Id not found.'));
     }
     if (count($form->_caseId) != 1) {
-      CRM_Core_Resources::fatal(ts('Expected one case-type'));
```

Comment:
the what?

Reply:
This sounds familiar. It comes from here as copy/paste - https://github.com/civicrm/civicrm-core/commit/eb20fbf0057966dded211ff822b98b52a159cfe1, but sort of makes sense. The actual issue is that _caseId became an array, so it was part of a larger change to only use the first one, and then a sanity check that there is only one when having more than one made no sense. But the wording is wrong since it has nothing to do with case types.

Reply:
Also they got the class wrong. `CRM_Core_Resources::fatal()` doesn't exist!